### PR TITLE
[DDoS Protection] Advanced TCP Protection filters (API instructions)

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-3056.

### CODE REVIEW

File: content/ddos-protection/tcp-protection/rule-settings.md [hunk 1]
```diff
@@ -6,25 +6,37 @@ weight: 9
 
 # Rule settings
 
-Each rule type has the following settings: mode, scope, burst sensitivity, and rate sensitivity.
+Each rule type has the following settings: scope, mode, burst sensitivity, and rate sensitivity.
 
 You may need to adjust the burst or rate sensitivity of a rule in case of false positives or due to specific traffic patterns.
 
+## Scope
```
Comment (pedrosousa) [inline content/ddos-protection/tcp-protection/rule-settings.md:13]: Moved content.

File: content/ddos-protection/tcp-protection/api/_index.md [hunk 2]
```diff
@@ -86,8 +88,25 @@ Delete all SYN flood rules | `DELETE accounts/<ACCOUNT_ID>/magic/advanced_tcp_pr
 
 {{</table-wrap>}}
 
+#### Filters
+
+{{<table-wrap>}}
+
+Operation | Method and endpoint / Description
```
Comment (kodster28) [inline content/ddos-protection/tcp-protection/api/_index.md:95]: Maybe for readability worth adding two `<br/>` after the api call? But that's just an opinion.

File: content/ddos-protection/tcp-protection/concepts.md [hunk 3]
```diff
@@ -24,3 +24,32 @@ The Advanced TCP Protection allowlist is a list of prefixes that will bypass all
...
+A filter allows you to modify Advanced TCP Protection's [execution mode](/ddos-protection/tcp-protection/rule-settings/#mode) — monitoring, mitigation (enabled), or disabled — for all incoming packets matching an expression. The expression can reference source and destination IP addresses and ports.
+
+Each type of advanced TCP protection (SYN flood protection and out-of-state TCP protection) has its own list of filters.
+
+{{<Aside type="warning" header="Important notes">}}
+* Currently, you can only manage filters [via API](/ddos-protection/tcp-protection/api/).
+* For each type of TCP protection (SYN flood protection or out-of-state TCP protection), you can only create one filter for each mode.
+{{</Aside>}}
+
+When you create more than one filter, their evaluation order is based on the filter's mode. A filter with `enabled` mode will be evaluated first (checking it for an expression match), then a filter with `monitoring` mode, and finally a filter with `disabled` mode.
```
Comment (kodster28) [inline content/ddos-protection/tcp-protection/concepts.md]: Might be worth reformatting this to be a numbered list? And / or integrating this info with the last section on the `Determining the execution mode`?

**Review by kodster28 — APPROVED** [review]

File: content/ddos-protection/tcp-protection/api/_index.md (hunk 2 above)
Comment (pedrosousa) [reply content/ddos-protection/tcp-protection/api/_index.md:95]: I'll try it, but I think the tables will become rather long (and we have quite a few tables here).

File: content/ddos-protection/tcp-protection/concepts.md (hunk 3 above)
Comment (pedrosousa) [reply content/ddos-protection/tcp-protection/concepts.md]: Yes, it makes sense to address this in the execution mode section. Thanks!

**Pull request merged**